### ISSUE TITLE
add ornament for OverlayerView make a new bug!!

### ISSUE THREAD

**Issue by roylly** [ephread/Instructions#168]
the example of “transition from code” scene, the ‘tap me’ button can't fire. As hitTest always return ornaments View, and the touch event will send to overlayView, the window under InstructionsWindow can't get the event!
I have found a solution:[ https://github.com/ephread/Instructions/pull/167](url)

**Comment by ephread** [comment]
Good catch 👍. The feature is a bit experimental and I hadn't tested everything. I'll take a look at the PR as soon as possible.
